specification/compute/resource-manager/Microsoft.Compute/stable/2022-03-01/ComputeRP/dedicatedHost.json

```diff
@@ -0,0 +1,1059 @@
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "DedicatedHosts_ListByHostGroup_MaximumSet_Gen": {
+            "$ref": "./examples/dedicatedHostExamples/DedicatedHosts_ListByHostGroup_MaximumSet_Gen.json"
+          },
+          "DedicatedHosts_ListByHostGroup_MinimumSet_Gen": {
+            "$ref": "./examples/dedicatedHostExamples/DedicatedHosts_ListByHostGroup_MinimumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/restart": {
+      "post": {
+        "tags": [
+          "DedicatedHost"
+        ],
+        "operationId": "DedicatedHosts_Restart",
+        "description": "Restart the dedicated host. The operation will complete successfully once the dedicated host has restarted and is running. To determine the health of VMs deployed on the dedicated host after the restart check the Resource Health Center in the Azure Portal. Please refer to https://docs.microsoft.com/en-us/azure/service-health/resource-health-overview for more details.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hostGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the dedicated host group."
+          },
+          {
+            "name": "hostName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the dedicated host."
+          },
+          {
+            "$ref": "../common.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../common.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../common.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Restart Dedicated Host.": {
+            "$ref": "./examples/dedicatedHostExamples/DedicatedHost_Restart.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "DedicatedHostGroupInstanceView": {
+      "properties": {
+        "hosts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DedicatedHostInstanceViewWithName"
+          },
+          "x-ms-identifiers": [],
```

Comment:
>"x-ms-identifiers": [] [](http://example.com/codeflow?start=10&length=22)

Looks like "name" was removed (as compared with 2021-11-01\compute.json. Was this intentional?

Reply:
this must have been missed. just synced it with Compute. Thank you @mentat9